core/src/test/java/com/backblaze/b2/client/B2StorageClientWebifierImplTest.java

```diff
@@ -1201,7 +1204,7 @@ public void testDownloadById() throws B2Exception {
                 "    X-Bz-Test-Mode: force_cap_exceeded\n"
         );
 
-        assertEquals(expectedUrl, webifier.getDownloadByIdUrl(ACCOUNT_AUTH, request));
+        assertEquals(expectedUrl, webifierSpy.getDownloadByIdUrl(ACCOUNT_AUTH, request));
```

Comment:
why does this need to use the spy'ed object?

Reply:
It doesn't need to be. Is it a good practice to only use the spy reference when using Mockito calls and the non spy in cases like this? I was thinking it might be less confusing if using the same spy reference throughout the code instead of mixing. Don't mind either way. I'll change it back